README.md

```diff
@@ -116,6 +116,30 @@ Another option is to have a remote rooms config file (this file needs to be acce
 		ROOMS_DATA=https://myfilelocation.io/myrooms_data.json
 
 
+#### Room groups
+
+You can also group rooms by addind `"group": "Group name"` in the setting of each room.
+
+Example:
+
+    {
```

Comment:
Hi @phiter What do you think we change the room structure to?

```
{
   group-id: "Group-1",
   name: "My Group Name"
   rooms: [
           {
               "id": "${UUID}",
               "name": "First room",
           },
          {
              "id": "${UUID}",
              "name": "Second room",
          },
          {
              "id": "${UUID}",
              "name": "Another room"
          }
   ]
}
```

We will break the current payload, but I think this is the best way to improve and simplify the way to manage rooms.

Reply:
Hi @juliemar, thanks for reviewing it.

I thought about that too, would probably be better but I wanted to make the change as small as possible, specially considering some people won't be using the grouping feature.

Changing the structure would probably need changes in many different places, specially the api which I didn't touch yet.

I suppose we could keep the grouping feature as it is now and maybe try something different in the future?